appinfo/routes.php

```diff
@@ -245,6 +245,22 @@
 				'apiVersion' => 'v2(\.1)?'
 			]
 		],
+		[
+			'name' => 'api#exportQuestion',
+			'url' => '/api/{apiVersion}/submissions/exportQuestion/{questionId}',
+			'verb' => 'GET',
+			'requirements' => [
+				'apiVersion' => 'v2'
```

Comment:
```suggestion
				'apiVersion' => 'v2.2'
```
Should only be available on 2.2 :)